src/selectors/entities/channels.js

```diff
@@ -764,3 +764,30 @@ export const filterPostIds = (condition) => {
         }
     );
 };
+
+const doGetProfilesInChannel = makeGetProfilesInChannel();
+const getOtherProfilesInChannel = (state) => {
+    return (channelId, currentUserId) => {
+        return doGetProfilesInChannel(state, channelId).filter((profile) => profile.id !== currentUserId);
+    };
+};
+export const getChannelsWithUserProfiles = createSelector(
+    getOtherProfilesInChannel,
```

Comment:
I'm not sure how familiar you are with [reselect selectors](https://github.com/reduxjs/reselect), but this one doesn't work correctly because `getOtherProfilesInChannel` returns a new function every time it's called, meaning that `getChannelsWithUserProfiles` recalculates its results each time it is ran. A better way to do this would be to get rid of `getOtherProfilesInChannel` and instead pass `getUserIdsInChannels` (which returns an object mapping channel ID to user IDs) and `getUsers` (which returns an object mapping user IDs to users).